tools/ci/check_grep.sh

```diff
@@ -82,6 +82,11 @@ if $grep -i 'nanotransen' $map_files; then
     echo -e "${RED}ERROR: Misspelling of Nanotrasen detected in maps, please remove the extra N(s).${NC}"
     st=1
 fi;
+if $grep 'NanoTrasen' $map_files; then
+	echo
+    echo -e "${RED}ERROR: Misspelling of Nanotrasen detected in maps, please uncapitalize the T(s).${NC}"
```

Comment:
```suggestion
    echo -e "${RED}ERROR: Misspelling of Nanotrasen detected in maps, please uncapitalize the T.${NC}"
```

there's only one T in Nanotrasen.

Reply:
I made it the same as the above `Misspelling of Nanotrasen detected in maps, please remove the extra N(s).`
It's referring to the fact that there can be multiple mispellings in the code that have to be removed, it doesn't mean a single example of it has multiple Ns/capital Ts.

Reply:
the grep error will spit itself out multiple times every time it finds an example of an error:
![image](https://user-images.githubusercontent.com/34697715/235334307-915454fa-ef05-481a-990f-bfc4ee8ce961.png)

via #67173

Reply:
or well it should, let me recheck

Reply:
Okay I'll fix the above one too then

Reply:
![image](https://user-images.githubusercontent.com/34697715/235334568-4c2c4bf0-6c42-4121-8bf4-aba2a60496bf.png)

ahhhh they changed it on me or i have dementia, either way nvm

Reply:
@BlueMemesauce lol revert that last commit it's my bad because the format makes it a lot more clear now

Reply:
Okay it doesn't, see https://github.com/BlueMemesauce/tgstation/actions/runs/4842310387/jobs/8629175983?pr=1

Reply:
sorry github is being slow to refresh unless you do it manually but yea